src/components/DropZone/DropZone.tsx

```diff
@@ -152,10 +152,13 @@ export class DropZone extends React.Component<CombinedProps, State> {
     return Object.keys(newState).length ? newState : null;
   }
 
-  private node: HTMLElement | null = null;
-  private dropNode: HTMLElement | HTMLDocument | null = null;
+  private node = React.createRef<HTMLDivElement>();
+  private dropNode = this.props.dropOnPage
```

Comment:
I don't think we need to use `createRef` here since it's never actually it's own ref. It's either document or `this.node`. This assignment will also only be ran once opposed to every render.

Reply:
Thank you for pointing that out! Yes, it would be quite redundant.